cmd/reviewdog/doghouse.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"strconv"
+	"strings"
+
+	"github.com/haya14busa/reviewdog"
+	"github.com/haya14busa/reviewdog/doghouse"
+	"github.com/haya14busa/reviewdog/doghouse/client"
+	"github.com/haya14busa/reviewdog/project"
+	"golang.org/x/sync/errgroup"
+)
+
+func runDoghouse(ctx context.Context, r io.Reader, installationID string, opt *option, isProject bool) error {
+	id, err := strconv.Atoi(installationID)
+	if err != nil {
+		return fmt.Errorf("installationID should be integer: %v, got %s", err, installationID)
+	}
+
+	ghInfo, isPr, err := getGitHubPR(opt.ci)
+	if err != nil {
+		return err
+	}
+	if !isPr {
+		return errors.New("this is not PullRequest build.")
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
error strings should not be capitalized or end with punctuation or a newline